static/rally/index.html

```diff
@@ -189,5 +189,22 @@
 </script>
 <!-- End Google Analytics -->
 
+<script>
+  var autocomplete;
+
+  function initAutocomplete() {
+    // Create the autocomplete object, restricting the search to geographical
+    // location types.
+    autocomplete = new google.maps.places.Autocomplete(
+        /** @type {!HTMLInputElement} */(document.getElementById('address_autocomplete')),
```

Comment:
yeah, it's actually stolen wholesale from the google maps autocomplete example — it seems to work, so the not breaking what isn't broke rule is what I followed ;-)